v4l2capture.c

```diff
@@ -47,8 +47,7 @@ struct buffer {
 };
 
 typedef struct {
-  PyObject_HEAD
-  int fd;
+  PyObject_HEAD int fd;
```

Comment:
This is probably an artifact left over after running the indent script on the code. Can you put the newline back?